android_docker_image.md

```diff
@@ -0,0 +1,109 @@
+The semaphore `android` docker image is a customized image based on `ubuntu:16.04`
+optimized for CI/CD. It comes with a set of preinstalled utility tools commonly
+used for CI/CD workflows.
+
+[[__TOC__]]
+
+## Using the semaphoreci/android docker image in your semaphore.yml configuration
+
+To use the images in this repository for your CI/CD pipelines, set up your Semaphore YAML with:
+
+``` yaml
+version: v1.0
+name: Hello andorid
+agent:
+  machine:
+    type: e1-standard-2
+  containers:
+    - name: main
+      image: semaphoreci/android:28
+blocks:
+  - name: "Hello"
+    task:
+      jobs:
+      - name: Hello
+        commands:
+          - checkout
+          - echo "Hello"
+```
+
+## Dockerhub repository
```

Comment:
Ideally this would be the very last section of this doc that only very brave and experienced people will need to pay attention to.